workflow/scripts/fgsea.R

```diff
@@ -0,0 +1,118 @@
+suppressPackageStartupMessages({
+  library("fgsea")
+  library("AnnotationDbi")
+})
```

Comment:
Please redirect all R script output to log files, see https://github.com/snakemake-workflows/rna-seq-star-deseq2/blob/master/scripts/deseq2-init.R#L1.

Reply:
https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/pull/4/commits/a9dbbb2efbbc4e7f5c759fa7e4d65611a71b2088

Reply:
Also did the same for the one Python script we have:
https://github.com/snakemake-workflows/rna-seq-kallisto-sleuth/pull/4/commits/ffbd8dbd976b8dd137fd3d202f202d198a9af7f5